data-access/burgemeester.ts

```diff
@@ -122,14 +143,18 @@ export const markCurrentBurgemeesterAsRejected = async (
     } ORDER BY DESC(?start) LIMIT 1
   `);
 
-  if (!result.results.bindings.length) {
+  const result = findFirstSparqlResult(findMandataris);
+  if (!result) {
     throw new HttpError(
       `No existing mandataris found for burgemeester(${burgemeesterUri})`,
       400,
     );
   }
-  const mandataris = result.results.bindings[0].mandataris.value;
-  const mandatarisUri = sparqlEscapeUri(mandataris);
+
+  endExistingMandataris(orgGraph, result.mandataris, date, benoeming);
+  // TODO: check use case if mandataris is waarnemend -> should something happen to the verhindering?
```

Comment:
The diagram in https://miro.com/app/board/uXjVMkUtg9A=/ says the verhindering should be ended, but do we want this in this call?
![Screenshot 2024-08-13 at 10 12 20](https://github.com/user-attachments/assets/08c4f861-0aaa-4c2c-bdda-8d6f0cf5e17a)

Reply:
let's check with Tim. making a note for next week